recipe/build.sh

```diff
@@ -75,6 +75,7 @@ python ./configure \
   --COPTFLAGS="$CFLAGS -O3" \
   --CXXOPTFLAGS="$CXXFLAGS -O3" \
   --FOPTFLAGS="$FFLAGS -O3" \
+  --CUDAOPTFLAGS="-O3" \
```

Comment:
this doesn't change behavior, but preserves default behavior while suppressing a warning